app/src/main/java/com/edu/uz/currency/currencyapp/retrofit/pojo/SingleCurrency.java

```diff
@@ -0,0 +1,54 @@
+
+package com.edu.uz.currency.currencyapp.retrofit.pojo;
+
+import java.util.List;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class SingleCurrency {
```

Comment:
Ten model może i jest okej, bo w takiej postaci przychodzi JSON i tak go trzeba odebrać, ale czy nie powinniśmy stworzyć jakiegoś naszego wewnętrznego modelu np. klasy Currency z polami: name, code, rate i ewentualnie data aktualizacji?  W sumie to chyba wszystko co jest nam potrzebne. Odczyt z całej tabeli byłby po prostu listą tych objektów Currency, bo nie widzę sensu, żeby były do tego chyba z 4 klasy, skoro z tego co widzę to jest to samo. Można zrobić mapper, który będzie to konwertował. Tak byłoby lepiej np. dlatego, że jak chcielibyśmy to trzymać w bazie to wystarczy jedna tabela, a nie 4.